### ISSUE TITLE
NTR: nasopharynx (example)

### ISSUE THREAD

**Issue by cmrn-rhi** [cidgoh/GRDI_AMR_One_Health#20]
<You may request multiple terms in a single issue. Delete the fields irrelevant to your request.>

### REQUIRED:
Associated Field(s): anatomical_part
Term Label: Nasopharynx
Definition/Description: The section of the pharynx that lies above the soft palate.

### OPTIONAL:
Definition Source: http://www.informatics.jax.org/accession/cwg
Synonyms or alternative labels: NP
Additional Comments (not an annotation): _description of usage within a sample plan_
Contributor Attribution: https://orcid.org/0000-0002-1825-0087

**Comment by cmrn-rhi** [comment]
This is a pretend/example new term request.
